### PR TITLE
added Surinder class

### PR DESCRIPTION
**Pull request by Surinder07**
Added Surinder.java class 
with   System.out.println(); statement 

TEST : 

we can run this method from intellij playbutton

### CODE REVIEW

File: src/Surinder.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Surinder {
+    public static void main(String[] args) {
+        System.out.println();
```
Comment (cpratham3) [inline src/Surinder.java:3]: can you add a message in print

Comment (khushi7899) [inline src/Surinder.java:3]: Add a hello message

**Review by khushi7899 — APPROVED** [review]

**Review by cpratham3 — APPROVED** [review]
review completed

File: src/Surinder.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+    }
```
Comment (Nishtha191) [inline src/Surinder.java:4]: it looks good

**Review by chintanghevariya — APPROVED** [review]

**Review by Lavishmakkar123 — APPROVED** [review]

File: src/Surinder.java (hunk 1 above)
Comment (Surinder07) [reply src/Surinder.java:3]: It looks fine to me

Comment (Surinder07) [reply src/Surinder.java:3]: I dont have time for that

**Pull request merged**